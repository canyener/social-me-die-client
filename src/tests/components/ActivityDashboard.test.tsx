// import React from 'react'
// import { shallow } from 'enzyme'

// import  ActivityDashboard  from '../../features/activities/dashboard/ActivityDashboard'
// import { fakeActivities } from '../fixtures/seed'

// test('Should render ActivityDashboard correctly with editMode equals to false', () => {
//   const wrapper = shallow(
//     <ActivityDashboard
//       activities={fakeActivities}
//       selectActivity={() => { }}
//       setEditMode={() => { }}
//       setSelectedActivity={() => { }}
//       createActivity={() => { }}
//       editActivity={() => { }}
//       deleteActivity={() => { }}
//       submitting={false}
//       target=''
//     />
//   )

//   expect(wrapper).toMatchSnapshot()
// })

// test('Should render ActivityDashboard correctly with editMode equals to true', () => {
//   const wrapper = shallow(
//     <ActivityDashboard
//       activities={fakeActivities}
//       selectActivity={() => { }}
//       setEditMode={() => { }}
//       setSelectedActivity={() => { }}
//       createActivity={() => { }}
//       editActivity={() => { }}
//       deleteActivity={() => { }}
//       submitting={false}
//       target=''
//     />
//   )

//   expect(wrapper).toMatchSnapshot()
// })

// test('Should render ActivityDashboard correctly with edit mode false and selected activity is null', () => {
//   const wrapper = shallow(
//     <ActivityDashboard
//       activities={fakeActivities}
//       selectActivity={() => { }}
//       setEditMode={() => { }}
//       setSelectedActivity={() => { }}
//       createActivity={() => { }}
//       editActivity={() => { }}
//       deleteActivity={() => { }}
//       submitting={false}
//       target=''
//     />
//   )

//   expect(wrapper).toMatchSnapshot()
// })

test('Will be refactored', () => {
  const message = 'Will be refactored after mobx store implementation'
  expect('Will be refactored after mobx store implementation').toBe(message)
})

export {}