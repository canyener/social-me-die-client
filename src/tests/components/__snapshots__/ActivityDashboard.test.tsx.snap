// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ActivityDashboard correctly with edit mode false and selected activity is null 1`] = `
<Grid>
  <GridColumn
    width={10}
  >
    <Component
      activities={
        Array [
          Object {
            "category": "Fake Category 1",
            "city": "Fake City",
            "date": "01.01.2020",
            "description": "Fake description",
            "id": "1",
            "title": "Fake Activity 1",
            "venue": "Fake Venue",
          },
          Object {
            "category": "Fake Category 2",
            "city": "Fake City 2",
            "date": "01.01.2020",
            "description": "Fake description 2",
            "id": "2",
            "title": "Fake Activity 2",
            "venue": "Fake Venue 2",
          },
        ]
      }
      selectActivity={[Function]}
    />
  </GridColumn>
  <GridColumn
    width={6}
  />
</Grid>
`;

exports[`Should render ActivityDashboard correctly with editMode equals to false 1`] = `
<Grid>
  <GridColumn
    width={10}
  >
    <Component
      activities={
        Array [
          Object {
            "category": "Fake Category 1",
            "city": "Fake City",
            "date": "01.01.2020",
            "description": "Fake description",
            "id": "1",
            "title": "Fake Activity 1",
            "venue": "Fake Venue",
          },
          Object {
            "category": "Fake Category 2",
            "city": "Fake City 2",
            "date": "01.01.2020",
            "description": "Fake description 2",
            "id": "2",
            "title": "Fake Activity 2",
            "venue": "Fake Venue 2",
          },
        ]
      }
      selectActivity={[Function]}
    />
  </GridColumn>
  <GridColumn
    width={6}
  >
    <Component
      activity={
        Object {
          "category": "Fake Category 1",
          "city": "Fake City",
          "date": "01.01.2020",
          "description": "Fake description",
          "id": "1",
          "title": "Fake Activity 1",
          "venue": "Fake Venue",
        }
      }
      setEditMode={[Function]}
      setSelectedActivity={[Function]}
    />
  </GridColumn>
</Grid>
`;

exports[`Should render ActivityDashboard correctly with editMode equals to true 1`] = `
<Grid>
  <GridColumn
    width={10}
  >
    <Component
      activities={
        Array [
          Object {
            "category": "Fake Category 1",
            "city": "Fake City",
            "date": "01.01.2020",
            "description": "Fake description",
            "id": "1",
            "title": "Fake Activity 1",
            "venue": "Fake Venue",
          },
          Object {
            "category": "Fake Category 2",
            "city": "Fake City 2",
            "date": "01.01.2020",
            "description": "Fake description 2",
            "id": "2",
            "title": "Fake Activity 2",
            "venue": "Fake Venue 2",
          },
        ]
      }
      selectActivity={[Function]}
    />
  </GridColumn>
  <GridColumn
    width={6}
  >
    <Component
      activity={
        Object {
          "category": "Fake Category 1",
          "city": "Fake City",
          "date": "01.01.2020",
          "description": "Fake description",
          "id": "1",
          "title": "Fake Activity 1",
          "venue": "Fake Venue",
        }
      }
      setEditMode={[Function]}
    />
  </GridColumn>
</Grid>
`;
